docs/contributors/code/auto-cherry-picking.md

```diff
@@ -0,0 +1,73 @@
+# Cherry-picking automation
+
+The cherry-picking automation makes cherry-picking PRs into WordPress and Gutenberg release branches a breeze.
+
+It works as follows: you switch to the relevant release branch and run  `npm run cherry-pick`.
+
+It gives you the following prompt:
+
+```
+You are on branch "wp/6.2".
```

Comment:
(After posting my other comments) The other command output samples all refer to WP 6.0 -- might be easier to change this instance to 

```suggestion
You are on branch "wp/6.0".
```

(and ignore all my other comments 😅 )